### PR TITLE
Log all output to both stderr and syslog.

### PR DESCRIPTION
**Pull request by squeed**
Because the tool is run just before a reboot, any docker-cached logs will be lost. So, log to syslog as well.

Fixes: OST-74

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ docker run \
     -v /etc/coreos:/etc/coreos:ro \
     -v /etc/torcx:/etc/torcx \
     -v /etc/kubernetes:/etc/kubernetes \
+	-v /dev/log:/dev/log \
```
Comment (lucab) [inline README.md]: Tabs vs spaces here?

File: cli/common.go [hunk 2]
```diff
@@ -35,6 +37,10 @@ var (
 
 // Init initializes the CLI environment for tectonic-torcx multicall
 func Init() error {
+	hook, err := logrus_syslog.NewSyslogHook("", "", syslog.LOG_INFO, "")
```
Comment (lucab) [inline cli/common.go]: I don't think I need this in the bootstrapper, output is already being logged via stdout->docker->journald. If it is feasible, can we move this to the hook only?

**Comment by squeed** [comment]
updated, PTAL

File: cli/tectonic-torcx-hook-pre.go [hunk 3]
```diff
@@ -43,6 +45,12 @@ func hookPreInit() {
 }
 
 func runHookPre(cmd *cobra.Command, args []string) error {
+	// Tee log output to syslog; docker logs are not peristed across reboots
+	// by default, so this hook may be very difficult to decode
```
Comment (lucab) [inline cli/tectonic-torcx-hook-pre.go]: Typo: "peristed"
I also think it was supposed to be s/decode/debug/.

**Comment by lucab** [comment]
LGTM. I haven't tested it as a CLUO pre-hook, but I will give it a spin before release.

**Pull request merged**